package.json

```diff
@@ -31,6 +31,7 @@
     "@babel/eslint-parser": "~7.17.0",
     "@babel/plugin-syntax-import-assertions": "~7.17.12",
     "@desertnet/html-parser": "~1.0.1",
+    "@types/node": "~17.0.34",
```

Comment:
I think this should be ^16, because that's the minimal required Node version in this project.

Reply:
I didn't realize the package's versions correlate with major NodeJS versions!